deepspeed/runtime/bf16_optimizer.py

```diff
@@ -341,7 +341,7 @@ def _update_hp_grad(self, lp, group_idx, param_idx, clear_lp_grads):
 
         # clear gradients
         if clear_lp_grads:
-            lp.grad._zero()
+            lp.grad.zero_()
```

Comment:
@inkcherry, will this break on older pytorch versions?

Reply:
It shouldn't, referring to the definition of ```zero_grad``` in version 1.10 at https://pytorch.org/docs/1.10/_modules/torch/optim/optimizer.html#Optimizer